pkgs/applications/virtualization/xen/generic/default.nix

```diff
@@ -25,7 +25,7 @@ versionDefinition:
   ncurses,
   ocamlPackages,
   perl,
-  python311Packages,
+  python3,
```

Comment:
Why did you switch from `python3Packages` to `python3`?
Using `python3Packages` has advantages in terms of splicing.
https://github.com/NixOS/nixpkgs/issues/211340

Reply:
Because xen is imported in python-packages.nix, and everyone there is writing `python3 = python` instead of `python3Packages = python.pkgs`, I was just following a convention

Reply:
Please use `python3Packages = self` instead.

Reply:
So it will be spliced the same way anyways?

Reply:
Done, thanks!

Reply:
TIL splicing, always wondered how does it work that used package is different whatevet it is put to buildInputs/nativeBuildInputs, assumed it somehow does `.override {stdenv = ...}`, but never checked.